### PR TITLE
CL-1212 Monitor tab dark theme

### PR DESCRIPTION
**Pull request by ianpaschal**

### CODE REVIEW

**Comment by ianpaschal** [comment]
screen shot 2019-01-22 at 10 56 16

🤷‍♂️

File: plugins/UM3NetworkPrinting/resources/qml/MonitorPrinterCard.qml [hunk 1]
```diff
@@ -94,15 +94,15 @@ Item
                 {
                     id: printerNameLabel
                     // color: "#414054" // TODO: Theme!
```
Comment (ImreBognarUltimaker) [inline plugins/UM3NetworkPrinting/resources/qml/MonitorPrinterCard.qml]: Is this line needed anymore?

File: plugins/UM3NetworkPrinting/resources/qml/MonitorPrinterCard.qml [hunk 2]
```diff
@@ -220,7 +220,7 @@ Item
         }
         border
         {
-            color: printer && printer.activePrintJob && printer.activePrintJob.configurationChanges.length > 0 ? "#f5a623" : "transparent" // TODO: Theme!
+            color: printer && printer.activePrintJob && printer.activePrintJob.configurationChanges.length > 0 ? UM.Theme.getColor("warning") : "transparent" // TODO: Theme!
             width: borderSize // TODO: Remove once themed
         }
         color: "transparent" // TODO: Theme!
```
Comment (ImreBognarUltimaker) [inline plugins/UM3NetworkPrinting/resources/qml/MonitorPrinterCard.qml]: Should this be put into the theme?

Comment (ianpaschal) [reply plugins/UM3NetworkPrinting/resources/qml/MonitorPrinterCard.qml]: Transparent not, as it just needs to not be visible sometimes and it's cleaner than to make two separate elements for it.

File: plugins/UM3NetworkPrinting/resources/qml/MonitorPrinterCard.qml (hunk 1 above)
Comment (ianpaschal) [reply plugins/UM3NetworkPrinting/resources/qml/MonitorPrinterCard.qml]: Good catch this one can be removed. Let me double check for comments.

Comment (ianpaschal) [reply plugins/UM3NetworkPrinting/resources/qml/MonitorPrinterCard.qml]: Fixed

File: plugins/UM3NetworkPrinting/resources/qml/MonitorPrintJobProgressBar.qml [hunk 3]
```diff
@@ -34,15 +34,15 @@ Item
         {
             background: Rectangle
             {
-                color: "#f5f5f5" // TODO: Theme!
-                implicitHeight: visible ? 8 * screenScaleFactor : 0 // TODO: Theme!
+                color: UM.Theme.getColor("monitor_progress_bar_empty")
+                implicitHeight: visible ? 12 * screenScaleFactor : 0 // TODO: Theme!
```
Comment (ImreBognarUltimaker) [inline plugins/UM3NetworkPrinting/resources/qml/MonitorPrintJobProgressBar.qml]: Is the change of this number intentional?

File: plugins/UM3NetworkPrinting/resources/qml/PrintJobContextMenu.qml [hunk 4]
```diff
@@ -18,14 +18,14 @@ Item {
     Button {
         id: button;
         background: Rectangle {
-            color: UM.Theme.getColor("viewport_background"); // TODO: Theme!
+            color: UM.Theme.getColor("monitor_card_hover"); // TODO: Theme!
```
Comment (ImreBognarUltimaker) [inline plugins/UM3NetworkPrinting/resources/qml/PrintJobContextMenu.qml]: Can the comment be removed now?

File: plugins/UM3NetworkPrinting/resources/qml/MonitorStage.qml [hunk 5]
```diff
@@ -36,24 +37,6 @@ Component
...
-                    color: "#f6f6f6" // TODO: Theme!
-                }
-                GradientStop
-                {
-                    position: 1.0
-                    color: "#ffffff" // TODO: Theme!
-                }
-            }
-        }
-
```
Comment (ImreBognarUltimaker) [inline plugins/UM3NetworkPrinting/resources/qml/MonitorStage.qml]: Is this deletion intentional?

File: plugins/UM3NetworkPrinting/resources/qml/MonitorPrintJobProgressBar.qml (hunk 3 above)
Comment (ianpaschal) [reply plugins/UM3NetworkPrinting/resources/qml/MonitorPrintJobProgressBar.qml]: Yup. Earlier designs of the monitor tab used a different progress bar style than the rest of Cura, this change brings it in line.

File: plugins/UM3NetworkPrinting/resources/qml/PrintJobContextMenu.qml (hunk 4 above)
Comment (ianpaschal) [reply plugins/UM3NetworkPrinting/resources/qml/PrintJobContextMenu.qml]: Yup. Good catch.

File: plugins/UM3NetworkPrinting/resources/qml/MonitorStage.qml (hunk 5 above)
Comment (ianpaschal) [reply plugins/UM3NetworkPrinting/resources/qml/MonitorStage.qml]: Yes, the gradient background did not work well with the fades on the carousel so we decided to drop them.

**Pull request merged**